### PR TITLE
CLI: Support TSX in storiesof-to-csf migration

### PR DESCRIPTION
**Pull request by shilman**
Issue: #9669

## What I did

Quick & dirty fix based on @Vanuan 's suggestion

## How to test

Run `storiesof-to-csf` migration on TSX project (untested)

### CODE REVIEW

File: lib/codemod/src/lib/utils.js [hunk 1]
```diff
@@ -13,3 +13,6 @@ export const sanitizeName = (name) => {
   }
   return key;
 };
+
+export const prettierParser = (parser) =>
```
Comment (Vanuan) [inline lib/codemod/src/lib/utils.js:17]: Maybe `jscodeshiftToPrettierParser`.
And use a map:
```
const parserMap = {
  'babylon': 'babel',
  'flow': 'flow',
  'ts': 'typescript',
  'tsx': 'typescript'
};

return parserMap[parser] || 'babel';
```

**Comment by ndelangen** [comment]
This doesn't add a test..

Can we add one and verify this actually works?

**Comment by Vanuan** [comment]
Is this merged? Is there a different PR?

**Comment by shilman** [comment]
@Vanuan no, i tried it out but there are more changes that need to happen and I don't have time to follow up

**Pull request closed without merging**